src/cohortMembers/dto/cohortMembers-search.dto.ts

```
@@ -19,11 +19,10 @@ enum SortDirection {
 }
 class FiltersDto {
   @ApiPropertyOptional({ type: String, description: 'Cohort ID', example: '' })
```

ESLint

[error] Replace `'Cohort·ID',·example:·''` with `"Cohort·ID",·example:·""`

(prettier/prettier)